bci_essentials/classification/erp_rg_classifier.py

```diff
@@ -368,19 +378,19 @@ def erp_rg_kernel(X, y):
         # Print performance stats
         # accuracy
         accuracy = sum(preds == self.y) / len(preds)
-        self.offline_accuracy = accuracy
+        self.offline_accuracy = [accuracy]
```

Comment:
Why the square brackets here?

Reply:
In the generic_classifier docs, offline_accuracy is defined as a list[float]. Here, it's calculated as a float, so I just had to explicitly encapsulate it in a list

Reply:
Makes sense, thanks!